desc/objectives/_equilibrium.py

```diff
@@ -103,29 +99,14 @@ def build(self, eq=None, use_jit=True, verbose=1):
         """
         eq = eq or self._eq
         if self._grid is None:
```

Comment:
So, if we run DESC from an input file, the equilibrium grid will be concentric?

Reply:
yes, `ConcentricGrid` with `jacobi` node pattern. This was already the default for all of our examples in `desc/examples` so shouldn't be any differences.

Reply:
The `Energy` objective defaults to a quadrature grid though.

Reply:
What is the difference between quadrature grid and jacobi grid? I thought jacobi node pattern was part of another quadrature grid.

Reply:
The main difference is the poloidal resolution. a `QuadratureGrid` uses 2*M+1 nodes on each surface to get exact poloidal integrals. The radial spacing is based on jacobi polynomials. a `ConcentricGrid` by default still uses jacobi spacing radially, but uses 2*M+1 nodes on the outermost surface, and fewer as you go inwards, usually down to 1 or 3 nodes near the axis. This is because we know things scale like rho**m so the theta variation near the axis is minimal, so we can get away with using roughly 1/2 as many nodes there. There's a more detailed explainer here: https://desc-docs.readthedocs.io/en/stable/notebooks/basis_grid.html#Collocation-grids